tools/attrd_updater.c

```diff
@@ -45,6 +45,8 @@ static struct crm_option long_options[] = {
     {"-spacer-",1, 0, '-', "\nCommands:"},
     {"update",  1, 0, 'U', "Update the attribute's value in attrd.  If this causes the value to change, it will also be updated in the cluster configuration"},
 #ifdef HAVE_ATOMIC_ATTRD
+    {"update-all",   1, 0, 'E', "Update the attribute's value, and the time to wait (dampening) in attrd.  If this causes the value to change, it will also be updated in the cluster configuration"},
+    {"update-dampen",   0, 0, 'F', "Update the time to wait (dampening) in attrd. If this causes the value to change, it will also be updated in the cluster configuration"},
```

Comment:
Okay!

I accept your suggestion.
I intend to change ATTRD_OP_UPDATE_DAMPEN to　ATTRD_OP_UPDATE_DELAY.